trappy/plotter/EventPlot.py

```diff
@@ -87,6 +87,16 @@ class EventPlot(AbstractDataPlotter):
 
         :param lanes: The sorted order of lanes
         :type lanes: list
+
+        :param color_map: A mapping between events and colours
+            ::
+                { "<name1>" : "colour1",
+                  .
+                  .
+                  .
+                  "<nameN>" : "colourN"
+                }
```

Comment:
Please document how you specify colors.  If by name, we could point to a table with names (coming from some HTML documentation?).  For hex, maybe put an example `#ff0000` is red or something like that.